packages/backend/server/src/modules/quota/types.ts

```diff
@@ -42,13 +42,13 @@ export type Quota = z.infer<typeof QuotaSchema>;
 
 @ObjectType()
 export class QuotaQueryType {
-  @Field(() => Int)
+  @Field(() => Float)
```

Comment:
Maybe we can use custom scalar
https://the-guild.dev/graphql/scalars/docs/scalars/safe-int

Reply:
replaced